values.yml

```diff
@@ -1,11 +1,8 @@
 #@data/values
 ---
-#@overlay/match missing_ok=True
```

Comment:
Why removing the missing_ok tokens here?

Reply:
Because they are defined in the `values-schema` in the `schema.yml` file